javascript/i18n/phonenumbers/phonenumberutil.js

```diff
@@ -743,47 +743,122 @@ i18n.phonenumbers.PhoneNumberUtil.VALID_PHONE_NUMBER_ =
+ // examples and this introduces many false interpretations as the extension labels
+ // are not standardized.
+ /** @type {string} */
+ var extLimitAfterExplicitLabel = '20';
+ /** @type {string} */
+ var extLimitAfterLikelyLabel = '15';
+ /** @type {string} */
+ var extLimitAfterAmbiguousChar = '9';
+ /** @type {string} */
+ var extLimitWhenNotSure = '6';
+
+ /** @type {string} */
+ var possibleSeparatorsBetweenNumberAndExtLabel = "[ \u00A0\\t,]*";
+ // Optional full stop (.) or colon, followed by zero or more spaces/tabs/commas.
+ /** @type {string} */
+ var possibleCharsAfterExtLabel = "[:\\.\uFF0E]?[ \u00A0\\t,-]*";
+ // Here the extension is called out in more explicit way, i.e mentioning it obvious
+ // patterns like "ext.".
+ /** @type {string} */
+ var explicitExtLabels =
+     "(?:e?xt(?:ensi(?:o\u0301?|\u00F3))?n?|\uFF45?\uFF58\uFF54\uFF4E?|\u0434\u043E\u0431|anexo)";
+ // One-character symbols that can be used to indicate an extension, and less
+ // commonly used or more ambiguous extension labels.
+ /** @type {string} */
+ var ambiguousExtLabels = "(?:[x\uFF58#\uFF03~\uFF5E]|int|\uFF49\uFF4E\uFF54)";
+ // When extension is not separated clearly.
+ /** @type {string} */
+ var ambiguousSeparator = "[- ]+";
+ // ",," is commonly used for auto dialling the extension when connected. First
+ // comma is matched through possibleSeparatorsBetweenNumberAndExtLabel, so we do
+ // not repeat it here. Semi-colon works in Iphone and Android also to pop up a
+ // button with the extension number following.
+ /** @type {string} */
+ var autoDiallingAndExtLabelsFound = "(?:,{2}|;)";
+ /** @type {string} */
+ var optionalExtnSuffix = "#?";
+
+ /** @type {string} */
+ var rfcExtn = i18n.phonenumbers.PhoneNumberUtil.RFC3966_EXTN_PREFIX_
+        + i18n.phonenumbers.PhoneNumberUtil.extnDigits_(extLimitAfterExplicitLabel);
+ /** @type {string} */
+ var explicitExtn = possibleSeparatorsBetweenNumberAndExtLabel + explicitExtLabels
+        + possibleCharsAfterExtLabel
+        + i18n.phonenumbers.PhoneNumberUtil.extnDigits_(extLimitAfterExplicitLabel)
+        + optionalExtnSuffix;
+ /** @type {string} */
+ var ambiguousExtn = possibleSeparatorsBetweenNumberAndExtLabel + ambiguousExtLabels
+        + possibleCharsAfterExtLabel
+	+ i18n.phonenumbers.PhoneNumberUtil.extnDigits_(extLimitAfterAmbiguousChar)
+	+ optionalExtnSuffix;
+ /** @type {string} */
+ var americanStyleExtnWithSuffix = ambiguousSeparator
+	+ i18n.phonenumbers.PhoneNumberUtil.extnDigits_(extLimitWhenNotSure) + "#";
+ // This is same as possibleSeparatorsBetweenNumberAndExtLabel, but not matching
+ // comma as extension label may have it.
+ /** @type {string} */
+ var possibleSeparatorsNumberExtLabelNoComma = "[ \u00A0\\t]*";
+
+ /** @type {string} */
+ var autoDiallingExtn = possibleSeparatorsNumberExtLabelNoComma
+        + autoDiallingAndExtLabelsFound + possibleCharsAfterExtLabel
+        + i18n.phonenumbers.PhoneNumberUtil.extnDigits_(extLimitAfterLikelyLabel)
+	+ optionalExtnSuffix;
```

Comment:
this + looks misaligned.

Reply:
Not able to view the issue. Maybe I have already corrected.

Reply:
In view of actual completed file, it is aligned rightly. :(